flake.nix

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Is this `flake.nix` something we can distribute to all your repos via `sbt-lucuma`?

Reply:
To be generated by sbt? I'm using it with direnv that expects the file to be there I think

More info:
https://github.com/typelevel/typelevel-nix

Reply:
Right, we can handle it like the github workflow and `.scalafmt.conf` files. It can be automatically generated and updated by sbt-lucuma, but still checked into the repository.

Reply:
Feel free to have a go. BTW the lock file probably shouldn't be generated